X00TD.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Ken_kaneki_69 (Paras Dhiman)",
+        "oem": "Asus",
+        "device": "Asus Zenfone Max Pro M1",
+        "filename": "crDroidAndroid-12.1-20220416-X00TD-v8.4.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/X00TD/8.x/crDroidAndroid-12.1-20220416-X00TD-v8.4.zip/download",
+        "timestamp": 1650131889,
+        "md5": "7aeb8195a78e3266212e1f556844d2a5",
+        "sha256": "240b55e7f86ce714259b04f1235cb5e7b45728519c4c727cbbd1a0618d365f53",
+        "size": 1099808016,
+        "version": "8.4",
+        "buildtype": "Monthly",
+        "forum": "https://t.me/mst_roms_X00T",
+        "gapps": "https://sourceforge.net/projects/nikgapps/files/Config-Releases/NikGapps-S/",
+        "firmware": "https://telegra.ph/Firmware-For-X00TD-Max-Pro-M1-06-07",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "https://forum.xda-developers.com/t/recovery-x00t-kudprojects-unofficial-twrp-3-5-2_9-1-k4-4-k4-14-16-05-2021.3888110/",
+        "paypal": "https://www.paypal.com/paypalme/luffymst?locale.x=en_GB",
+        "telegram": "https://t.me/mst_roms_X00T",
+        "dt": "https://github.com/crdroidandroid/android_device_asus_X00TD",
+        "common-dt": "https://github.com/crdroidandroid/android_device_asus_sdm660-common",
```

Comment:
encryption needs to be enabled by default as per new rules
https://github.com/crdroidandroid/rules-and-guidelines#hardware - see storage
https://github.com/crdroidandroid/android_device_asus_sdm660-common/blob/cr/rootdir/etc/fstab.qcom#L14#L15

also try setting branch to 12.1 rather then cr